plugins/org.preesm.codegen.xtend/xtend-src/org/preesm/codegen/xtend/printer/c/CHardwarePrinter.xtend

```diff
@@ -138,23 +138,23 @@ class CHardwarePrinter extends CPrinter {
 
 	override generateStandardLibFiles() {
 		val result = super.generateStandardLibFiles();
-		val String stdFileFolderHardware = "/stdfiles/hardware/"
-		val filesHardware = Arrays.asList(#[
-			"hardware.c",
-			"hardware.h",
-			"hardware_hw.c",
-			"hardware_hw.h",
-			"hardware_rcfg.c",
-			"hardware_rcfg.h",
-			"hardware_dbg.h"
-		]);
-		filesHardware.forEach [ it |
-			try {
-				result.put(it, PreesmResourcesHelper.instance.read(stdFileFolderHardware + it, this.class))
-			} catch (IOException exc) {
-				throw new PreesmRuntimeException("Could not generated content for " + it, exc)
-			}
-		]
+//		val String stdFileFolderHardware = "/stdfiles/hardware/"
+//		val filesHardware = Arrays.asList(#[
+//			"hardware.c",
+//			"hardware.h",
+//			"hardware_hw.c",
+//			"hardware_hw.h",
+//			"hardware_rcfg.c",
+//			"hardware_rcfg.h",
+//			"hardware_dbg.h"
+//		]);
+//		filesHardware.forEach [ it |
+//			try {
+//				result.put(it, PreesmResourcesHelper.instance.read(stdFileFolderHardware + it, this.class))
+//			} catch (IOException exc) {
+//				throw new PreesmRuntimeException("Could not generated content for " + it, exc)
+//			}
+//		]
```

Comment:
Please remove the code instead of commenting it.

Reply:
The whole function was removed as well as the folder containing the template of the printed files. The original `super` class function is now used.